ansible/roles/kong-api/defaults/main.yml

```diff
@@ -354,6 +354,25 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: compositePrivateSearch
+    uris: "{{ composite_service_prefix }}/v1/private/search"
+    upstream_url: "{{ knowledge_mw_service_url }}/v3/private/search"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: 
+        - 'contentAdmin'
+        - 'contentTempAccess'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ premium_consumer_large_rate_limit_per_hour }}"
```

Comment:
Please state why we need a premium rate limit for this? Who are the end users? How many calls can happen over an hour?